SQL/DQL/union/union.md

```diff
@@ -0,0 +1,128 @@
+# UNION
+author: mihaiberq
+
+levels:
+  - beginner
+  - basic
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - workout
+
+---
+## Content
+
+The **union** of two or more tables means combining their **type-matching rows**. Unlike the `JOIN` operation, which combines columns (containing all entries from either the left, the right or both tables), the `UNION` operation always retrieves every entry in *both tables*.
```

Comment:
This really needs an illustration, and some example results

Reply:
the example results come later in the insight. I will try to think of an illustration and kindly ask Matt to draw it :+1: